blog/models.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.db.models import Count
+from django.urls import reverse
+
+from blog_system import settings
+
+
+class User(AbstractUser):
+    email = models.CharField(max_length=63)
+    first_name = models.CharField(max_length=63)
+    last_name = models.CharField(max_length=63)
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(settings.AUTH_USER_MODEL,
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line